src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -1503,7 +1505,11 @@ private static ConformanceConfig loadConformanceConfig(String configFile)
    * within the directory and sub-directories.
    */
   public static List<String> findJsFiles(Collection<String> patterns) throws IOException {
-    Set<String> allJsInputs = new TreeSet<>();
+    return findJsFiles(patterns, new LinkedHashSet<String>());
+  }
+
+  private static List<String> findJsFiles(Collection<String> patterns, Set<String> allJsInputs)
```

Comment:
This should probably have some JavaDoc, particularly since we're adding things to a collection that is passed as a param.